org.eclipse.winery.repository.ui/src/app/instance/sharedComponents/properties/properties.component.ts

```diff
@@ -75,10 +72,11 @@ export class PropertiesComponent implements OnInit {
             this.properties = data.properties;
         } else {
             this.isXMLData = false;
-            if (!isNullOrUndefined(data.properies)) {
+            if (!isNullOrUndefined(data.properties)) {
                 this.propertyKeys = Object.keys(data.properties);
+                this.propertyKeys.sort();
```

Comment:
Is this really necessary? I think, it does not adhere the ordering configured at the type.

Reply:
Maybe, this line can just be removed and the backend implementation needs to be modified in a follow-up PR.